### DISCUSSION TITLE
AR View Stuck in loading

### DISCUSSION THREAD

**Discussion by heikog** [google/model-viewer#3488] [Q&A]
### Description

I included 2 models on my webpage. The first one is working fine.
For the second: viewing the model is fine - but switching to AR mode makes the viewer being stuck in some loading animation

#### Live Demo
https://kulturschokolade.org/ar

### Version

latest

<model-viewer bounds="tight" src="https://kulturschokolade.org/wp-content/uploads/3DContent/ReadyPlayerMe.glb" ar ar-modes="scene-viewer" camera-controls environment-image="neutral" shadow-intensity="1" camera-orbit="0deg 90deg auto">
</model-viewer>

<model-viewer bounds="tight" enable-pan src="https://kulturschokolade.org/wp-content/uploads/3DContent/DieTrauernde.glb" ar ar-modes="scene-viewer" camera-controls shadow-intensity="1" camera-orbit="-171.7deg 90deg auto">

   
</model-viewer>

### Browser Affected
IOS 15.4 Safari

### AR

- [ ] WebXR
- [X] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
Scene Viewer isn't an option on iOS. Do you mean Quick Look?

**Comment by amcc** [comment]
this might be related - but putting multiple models on one page means i can't open the second model - also on iOS Safari for me.

For me i see the AR button on both models - when i click on the AR button on the first model it behaves normally, if i click on the AR button on the second model the AR button simply disappears and the AR does not get triggered - i can't get that button back without refreshing.

**Reply by elalish** [reply]
I'm not sure they are related. First, ensure you have Safari 15.5, because 15.4 had some major bugs. Are you sure this is about two models on the page, or about a particular model? Try posting a simple Glitch repro link so we can see the source of the problem.
